zgw_consumers/admin_fields.py

```diff
@@ -45,21 +49,42 @@ def get_zaaktypen() -> Dict[Service, List[Dict[str, Any]]]:
 
 
 def get_zaaktype_field(db_field: Field, request: HttpRequest, **kwargs):
-    zaaktypen = get_zaaktypen()
-
-    def _get_choice(zaaktype: dict) -> Tuple[str, str]:
-        return (
-            zaaktype["url"],
-            f"{zaaktype['identificatie']} - {zaaktype['omschrijving']}",
+    try:
+        zaaktypen = get_zaaktypen()
+    except ClientError as exc:
+        error_json = exc.args[0]
+        messages.error(
+            request,
+            _(
+                "Failed to retrieve available zaaktypen "
+                "(got {http_status} - {detail}). "
+                "The cause of this exception was: {cause}"
+            ).format(
+                http_status=error_json["status"],
+                detail=error_json["detail"],
+                cause=exc.__cause__,
+            ),
         )
+        choices = []
+    except HTTPError as exc:
+        error_json = exc.args[0]
+        choices = []
+        messages.error(request, f"{error_json}")
```

Comment:
`error_json` is a bad name here, since `exc.args[0]` is a string that is built inside of `requests` with some information. `error_message` would be better.